model2.py

```diff
@@ -100,16 +110,23 @@ def model(self, zero_data, covariates):
 
         rate = scale.reciprocal()
         concentration = mean * rate
-        # alternative: GammaPoisson (or NegativeBinomial)
+        # alternative: GammaPoisson (or NegativeBinomial, ZeroInflatedNegativeBinomial)
         noise_dist = dist.Gamma(concentration, rate)
 
         with store_plate, product_plate:
             self.predict(noise_dist, mean.new_zeros(mean.shape))
 
 
+# By default, the Forecaster class will use the AutoNormal guide by default.
+# It saves time to construct a guide for our model. However, in some situations,
+# constructing a custom guide is helpful (especially for armotized guides depending
```

Comment:
typo: armotized

Reply:
whoa, you have good eyes ;) thanks for detailed reviewing!